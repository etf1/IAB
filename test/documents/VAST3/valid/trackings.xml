<!-- <Creative>s with all differents <TrackingEvents> -->
<VAST version="3.0">
    <Ad id="601364">
        <InLine>
            <AdSystem>Acudeo Compatible</AdSystem>
            <AdTitle>VAST 2.0 Instream Test 1</AdTitle>
            <Description>VAST 2.0 Instream Test 1</Description>
            <Error>http://myErrorURL/error</Error>

            <Impression>http://myTrackingURL/impression</Impression>
            <Creatives>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <TrackingEvents>
                            <Tracking event="creativeView">http://www.google.com</Tracking>
                            <Tracking event="creativeView">
                                http://www.google.com
                            </Tracking>
                            <Tracking event="start"><![CDATA[http://www.google.com]]></Tracking>
                            <Tracking event="firstQuartile">
                                <![CDATA[http://www.google.com]]>
                            </Tracking>
                            <Tracking event="midpoint">http://www.google.com</Tracking>
                            <Tracking event="thirdQuartile">http://www.google.com</Tracking>
                            <Tracking event="complete">http://www.google.com</Tracking>
                            <Tracking event="mute">http://www.google.com</Tracking>
                            <Tracking event="unmute">http://www.google.com</Tracking>
                            <Tracking event="pause">http://www.google.com</Tracking>
                            <Tracking event="rewind">http://www.google.com</Tracking>
                            <Tracking event="resume">http://www.google.com</Tracking>
                            <Tracking event="fullscreen">http://www.google.com</Tracking>
                            <Tracking event="exitFullscreen">http://www.google.com</Tracking>
                            <Tracking event="expand">http://www.google.com</Tracking>
                            <Tracking event="collapse">http://www.google.com</Tracking>
                            <Tracking event="acceptInvitation">http://www.google.com</Tracking>
                            <Tracking event="close">http://www.google.com</Tracking>
                            <Tracking event="skip">http://www.google.com</Tracking>
                            <Tracking event="progress" offset="30%">http://www.google.com</Tracking>
                            <Tracking event="progress" offset="5.2563%">http://www.google.com</Tracking>
                            <Tracking event="progress" offset="00:01:20">http://www.google.com</Tracking>
                            <Tracking event="progress" offset="15:20:59.123">http://www.google.com</Tracking>
                        </TrackingEvents>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <TrackingEvents>
                            <Tracking event="progress" offset="15:20:59.123">http://www.google.com</Tracking>
                        </TrackingEvents>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <TrackingEvents></TrackingEvents>
                    </Linear>
                </Creative>
            </Creatives>
        </InLine>
    </Ad>
</VAST>
