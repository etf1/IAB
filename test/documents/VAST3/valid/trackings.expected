{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "VAST 2.0 Instream Test 1",
      "description": "VAST 2.0 Instream Test 1",
      "adSystem": {
        "name": "Acudeo Compatible"
      },
      "id": "601364",
      "error": "http://myErrorURL/error",
      "impressions": [
        {
          "uri": "http://myTrackingURL/impression"
        }
      ],
      "creatives": [
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "trackings": [
            {
              "event": 1,
              "uri": "http://www.google.com"
            },
            {
              "event": 1,
              "uri": "http://www.google.com"
            },
            {
              "event": 2,
              "uri": "http://www.google.com"
            },
            {
              "event": 3,
              "uri": "http://www.google.com"
            },
            {
              "event": 4,
              "uri": "http://www.google.com"
            },
            {
              "event": 5,
              "uri": "http://www.google.com"
            },
            {
              "event": 6,
              "uri": "http://www.google.com"
            },
            {
              "event": 7,
              "uri": "http://www.google.com"
            },
            {
              "event": 8,
              "uri": "http://www.google.com"
            },
            {
              "event": 9,
              "uri": "http://www.google.com"
            },
            {
              "event": 10,
              "uri": "http://www.google.com"
            },
            {
              "event": 11,
              "uri": "http://www.google.com"
            },
            {
              "event": 12,
              "uri": "http://www.google.com"
            },
            {
              "event": 13,
              "uri": "http://www.google.com"
            },
            {
              "event": 14,
              "uri": "http://www.google.com"
            },
            {
              "event": 15,
              "uri": "http://www.google.com"
            },
            {
              "event": 16,
              "uri": "http://www.google.com"
            },
            {
              "event": 17,
              "uri": "http://www.google.com"
            },
            {
              "event": 18,
              "uri": "http://www.google.com"
            },
            {
              "event": 19,
              "uri": "http://www.google.com",
              "offset": "30%"
            },
            {
              "event": 19,
              "uri": "http://www.google.com",
              "offset": "5.2563%"
            },
            {
              "event": 19,
              "uri": "http://www.google.com",
              "offset": "00:01:20"
            },
            {
              "event": 19,
              "uri": "http://www.google.com",
              "offset": "15:20:59.123"
            }
          ]
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "trackings": [
            {
              "event": 19,
              "uri": "http://www.google.com",
              "offset": "15:20:59.123"
            }
          ]
        },
        {
          "creativeType": 1,
          "duration": "01:00:00"
        }
      ]
    }
  ]
}
